src/AutoSysimages.jl

```diff
@@ -389,36 +393,44 @@ end
 This install the `asysimg` scripts.
 (Currently implemented only for Linux.)
 """
-function install(dir=_default_install_dir())
+function install(dir = _default_install_dir())
+
+    dir_exists = ispath(dir)
+    dir_exists || mkpath(dir)
+
     if Sys.islinux() || Sys.isapple()
-        dir_exists = ispath(dir)
-        dir_exists || mkpath(dir)
         script = joinpath(@__DIR__, "..", "scripts", "linux", "asysimg")
         script = abspath(normpath(script))
-        cp(script, joinpath(dir, "asysimg"), force=true)
+        cp(script, joinpath(dir, "asysimg"), force = true)
+    elseif Sys.iswindows()
+        script = joinpath(@__DIR__, "..", "scripts", "windows", "asysimg.bat")
```

Comment:
This is kind of duplicite code for Linux and Windows (except for the extension). We can merge together.